FantasmoSDK/app/src/main/java/com/fantasmo/sdk/models/analytics/TotalDeviceRotationAccumulator.kt

```diff
@@ -47,34 +52,33 @@ class TotalDeviceRotationAccumulator {
      * @param rotation FloatArray correspondent to the rotationQuaternion
      */
     private fun updateRotationValues(rotation: FloatArray) {
-        val rads = convertQuaternionToEuler(rotation)
-        val yawCurrent = rads[0]
-        val pitchCurrent = rads[1]
-        val rollCurrent = rads[2]
+        val yawCurrent = rotation[0]
+        val pitchCurrent = rotation[1]
+        val rollCurrent = rotation[2]
 
         //Update Yaw values
         yaw[0] = min(yawCurrent, yaw[0])
         yaw[1] = max(yawCurrent, yaw[1])
-        yaw[2] = min(max(yaw[1] - yaw[0], 0f), 2 * Math.PI.toFloat())
+        yaw[2] = min(max(yaw[1] - yaw[0], 0f), 360f)
 
         //Update Pitch values
         pitch[0] = min(pitchCurrent, pitch[0])
         pitch[1] = max(pitchCurrent, pitch[1])
-        pitch[2] = min(max(pitch[1] - pitch[0], 0f), 2 * Math.PI.toFloat())
+        pitch[2] = min(max(pitch[1] - pitch[0], 0f), 360f)
 
         //Update Roll values
         roll[0] = min(rollCurrent, roll[0])
         roll[1] = max(rollCurrent, roll[1])
-        roll[2] = min(max(roll[1] - roll[0], 0f), 2 * Math.PI.toFloat())
+        roll[2] = min(max(roll[1] - roll[0], 0f), 360f)
     }
 
     /**
      * Resets counters and each axis rotation array
      */
     fun reset() {
         frameCounter = 0
-        yaw = floatArrayOf(0f, 0f, 0f)
-        pitch = floatArrayOf(0f, 0f, 0f)
-        roll = floatArrayOf(0f, 0f, 0f)
+        yaw = floatArrayOf(Float.POSITIVE_INFINITY, Float.NEGATIVE_INFINITY, 0f)
+        pitch = floatArrayOf(Float.POSITIVE_INFINITY, Float.NEGATIVE_INFINITY, 0f)
```

Comment:
I still think this needs to reset to -90, +90 (I think that's how it's handled on iOS)